### PR TITLE
Add a &nbsp; after timestamp

### PR DESCRIPTION
**Pull request by rubo77**
This solves part of the problem that the timestamp is  gluing at the message, when copied in your clipboard.

Example:

    23:02test message

see https://github.com/vector-im/riot-web/issues/893

Signed-off-by: Ruben Barkow <github@r.z11.de>

### CODE REVIEW

**Comment by ara4n** [comment]
what browser does this problem occur on? i don't see the timestamp glued to the message when copying from chrome...

**Comment by rubo77** [comment]
Yes, in Chrome it seems fine, in Firefox it is glued.

**Comment by dbkr** [comment]
Sorry again that this has got missed for almost a year. It's against the wrong branch though and you need to do the DCO signoff.

**Comment by rubo77** [comment]
Can't you please just cherry pick it from here?

**Comment by t3chguy** [comment]
@rubo77 you can change the branch at the top of the pull request, the more important bit is the DCO

**Comment by rubo77** [comment]
I changed to master.

Hereby I sign the DCO

**Comment by t3chguy** [comment]
It should be on `develop`. Which is why I changed it to that...

**Comment by t3chguy** [comment]
@rubo77 please read the Contributing doc properly: https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#sign-off

**Comment by rubo77** [comment]
Sorry, I'm only on mobile until Monday

**Comment by rubo77** [comment]
Is the sign off on the first post correct?

File: src/components/views/messages/MessageTimestamp.js [hunk 1]
```diff
@@ -29,8 +29,10 @@ module.exports = React.createClass({
     render: function() {
         const date = new Date(this.props.ts);
         return (
+            <span className="mx_MessageTimestamp">
+                { DateUtils.formatTime(date)+'&nbsp;' }
             <span className="mx_MessageTimestamp" title={ DateUtils.formatFullDate(date, this.props.showTwelveHour) }>
```
Comment (t3chguy) [inline src/components/views/messages/MessageTimestamp.js]: Looks like a bad merge @dbkr

Comment (dbkr) [reply src/components/views/messages/MessageTimestamp.js]: oops, good spot

**Pull request merged**